stephanie-smith/short-response.md

```diff
@@ -0,0 +1,66 @@
+# Problem Set 2.2 - Variables and Control Flow
+## Short Response Questions
+
+1. **What does the code below log? Why?**
+   ```javascript
+    {
+      let singer = 'Omar Apollo';
+    }
+
+    console.log(`My favorite singer is ${singer}`);
+   ```
+   - It logs "My favorite singer is Omar Apollo".
```

Comment:
Check this again. Run it in the console. Be sure to tell me _why_ it does what it does.